docs/scroll-into-viewing-area.md

```diff
@@ -1,3 +1,48 @@
-# Scroll Into Viewing Area
+# Scroll Into View
```

Comment:
Should we rename this file to `scroll-into-view.md`?

Reply:
yeeeep.  thought I had done that :joy